lib/partiql-transpiler/src/test/resources/targets/redshift/redshift_results_00.ion

```diff
@@ -56,19 +56,46 @@ target::{
     },
     '0004': {
       statement: '''
-        SELECT
-          store.s_store_sk AS s_store_sk
-        FROM
-            tpc_ds."store" AS store
-          LEFT OUTER JOIN
-            tpc_ds."store_returns" AS returns
-          ON store.s_store_sk = returns.sr_store_sk
+        SELECT store.s_store_sk AS s_store_sk, store.s_store_id AS s_store_id, store.s_rec_start_date AS s_rec_start_date,
+        store.s_rec_end_date AS s_rec_end_date, store.s_closed_date_sk AS s_closed_date_sk, store.s_store_name AS s_store_name,
+        store.s_number_employees AS s_number_employees, store.s_floor_space AS s_floor_space, store.s_hours AS s_hours,
+        store.s_manager AS s_manager, store.s_market_id AS s_market_id, store.s_geography_class AS s_geography_class,
+        store.s_market_desc AS s_market_desc, store.s_market_manager AS s_market_manager, store.s_division_id AS s_division_id,
+        store.s_division_name AS s_division_name, store.s_company_id AS s_company_id, store.s_company_name AS s_company_name,
+        store.s_street_number AS s_street_number, store.s_street_name AS s_street_name, store.s_street_type AS s_street_type,
+        store.s_suite_number AS s_suite_number, store.s_city AS s_city, store.s_county AS s_county, store.s_state AS s_state,
+        store.s_zip AS s_zip, store.s_country AS s_country, store.s_gmt_offset AS s_gmt_offset,
+        store.s_tax_precentage AS s_tax_precentage, returns.sr_returned_date_sk AS sr_returned_date_sk,
+        returns.sr_return_time_sk AS sr_return_time_sk, returns.sr_item_sk AS sr_item_sk,
+        returns.sr_customer_sk AS sr_customer_sk, returns.sr_cdemo_sk AS sr_cdemo_sk, returns.sr_hdemo_sk AS sr_hdemo_sk,
+        returns.sr_addr_sk AS sr_addr_sk, returns.sr_store_sk AS sr_store_sk, returns.sr_reason_sk AS sr_reason_sk,
+        returns.sr_ticket_number AS sr_ticket_number, returns.sr_return_quantity AS sr_return_quantity,
+        returns.sr_return_amt AS sr_return_amt, returns.sr_return_tax AS sr_return_tax,
+        returns.sr_return_amt_inc_tax AS sr_return_amt_inc_tax, returns.sr_fee AS sr_fee,
+        returns.sr_return_ship_cost AS sr_return_ship_cost, returns.sr_refunded_cash AS sr_refunded_cash,
+        returns.sr_reversed_charge AS sr_reversed_charge, returns.sr_store_credit AS sr_store_credit,
+        returns.sr_net_loss AS sr_net_loss
+        FROM tpc_ds."store" AS store
+        LEFT OUTER JOIN tpc_ds."store_returns" AS returns
+        ON store.s_store_sk = returns.sr_store_sk
```

Comment:
I kind of think we should use a different suite for `SELECT *` with tables that have few columns just so this isn't so large